### ISSUE TITLE
Standard arguments for convert/generate scripts

### ISSUE THREAD

**Issue by SamPUG** [openmc-dev/data#18]
This issue is added in response to a requirement from @Shimwell and @ukaea.

There are currently different arguments used in each of the scripts. Some of them use `--download` while others use `--batch`. The arguments for each script should be standardised as discussed in #8. 

I am planning to replace the `--batch` argument in all relevant scripts with the same argument format as the JEFF scripts, eg:
```
parser.add_argument('--download', action='store_true',  
                    help='Download files from OECD-NEA')
parser.add_argument('--no-download', dest='download', action='store_false',
                    help='Do not download files from OECD-NEA')
parser.add_argument('--extract', action='store_true',
                    help='Extract tar/zip files')
parser.add_argument('--no-extract', dest='extract', action='store_false',
                    help='Do not extract tar/zip files')
```

**Comment by Shimwell** [comment]
Yes please this would be great as it is a bit of inconstant at the moment

Once this is done I might even be tempted to update the readme :-)

**Comment by paulromano** [comment]
Closed with #26
